### PR TITLE
add hacktoberfest to readme

### PR DESCRIPTION
**Pull request by madmas**
Add Hacktoberfest Logo to readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,11 @@ This repository contains all Java related sources of Rico. Clients for other lan
 
 ![Rico Logo](readme/rico_logo.png)
 
+![Hacktoberfest Logo](https://hacktoberfest.digitalocean.com/assets/HF-full-logo-b05d5eb32b3f3ecc9b2240526104cf4da3187b8b61963dd9042fdc2536e4a76c.svg)
+
+*This project joined [Hacktoberfest](https://hacktoberfest.digitalocean.com/)!
```
Comment (sclassen) [inline README.md]:
```suggestion
*This project joined [Hacktoberfest 2020](https://hacktoberfest.digitalocean.com/)!
```

**Review by sclassen — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -8,6 +8,11 @@ This repository contains all Java related sources of Rico. Clients for other lan
...
+Issues that can give you a good start are [have the label "Hacktoberfest"](https://github.com/rico-projects/rico/issues?q=is%3Aissue+is%3Aopen+label%3A%22Hacktoberfest%22) and maintainers are willing to help.*
```
Comment (hendrikebbers) [inline README.md:14]: As far as I know it is "hacktoberfest" label and not "Hacktoberfest". Does this make a difference?

**Comment by hendrikebbers** [comment]
Another example of such addition that contains more descriptions: https://gist.github.com/hendrikebbers/0f9cdd18076343b3bbe5f2d162733b6e

File: README.md (hunk 2 above)
Comment (madmas) [reply README.md:14]: No, it doesnt - but the issues in our repo are actually uppercase ;-)

**Comment by madmas** [comment]
@sclassen @hendrikebbers pls merge if you approve

**Pull request merged**